rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
```

Comment:
Will this file be used?

Reply:
I do hope so. Is your concern that the file is not in the same directory as the Cargo.toml files?
The [doc](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file) is a bit ambiguous, but I do hope it gets used!

Reply:
> Is your concern that the file is not in the same directory as the Cargo.toml files?

precisely. I am not sure either.

Reply:
I guess the answer is yes, otherwise CI would have failed :-)